examples/seismic/tti/tti_example.py

```diff
@@ -64,20 +58,24 @@ def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
                         type=int, help="Time order of the simulation")
     parser.add_argument("-so", "--space_order", default=4,
                         type=int, help="Space order of the simulation")
-    parser.add_argument("--nbpml", default=10,
+    parser.add_argument("--nbpml", default=40,
                         type=int, help="Number of PML layers around the domain")
+    parser.add_argument("-k", dest="kernel", default='centered',
+                        type=str, help="Choice of finite-difference kernel")
```

Comment:
If it's a choice, you should make that explicit with `choices=['a', 'b', 'c']` instead of `type=str`. This way the script documentation will list the possible options.